near-sdk/src/environment/env.rs

```diff
@@ -956,10 +1742,34 @@ pub fn storage_remove(key: &[u8]) -> bool {
     }
 }
 /// Reads the most recent value that was evicted with `storage_write` or `storage_remove` command.
+///
+/// # Use cases
+/// Storage functions are typically used to upgrade/migrate a contract state, preventing errors like `Cannot deserialize the contract state` after rolling out the breaking changes to the network.
```

Comment:
this call isn't used in the referenced repo, and also in a pending pr https://github.com/near-examples/update-migrate-rust/pull/2 
won't request change to remove this link, hoping that this and some of other suggestions will be adressed in some of subsequent prs